en/core-libraries/time.rst

```diff
@@ -298,86 +352,96 @@ Comparing With the Present
 .. php:method:: isThisMonth()
 .. php:method:: isThisYear()
 
-You can compare a ``Time`` instance with the present in a variety of ways::
+You can compare a ``FrozenTime`` instance with the present in a variety of ways::
 
-    $time = new Time('2014-06-15');
+    $time = new FrozenTime('+3 days');
 
-    echo $time->isYesterday();
-    echo $time->isThisWeek();
-    echo $time->isThisMonth();
-    echo $time->isThisYear();
+    var_export($time->isYesterday());
+    var_export($time->isThisWeek());
+    var_export($time->isThisMonth());
+    var_export($time->isThisYear());
 
 Each of the above methods will return ``true``/``false`` based on whether or
-not the ``Time`` instance matches the present.
+not the ``FrozenTime`` instance matches the present.
 
 Comparing With Intervals
 ========================
 
 .. php:method:: isWithinNext($interval)
 
-You can see if a ``Time`` instance falls within a given range using
+You can see if a ``FrozenTime`` instance falls within a given range using
 ``wasWithinLast()`` and ``isWithinNext()``::
 
-    $time = new Time('2014-06-15');
+    $time = new FrozenTime('+3 days');
 
-    // Within 2 days.
-    echo $time->isWithinNext(2);
+    // Within 2 days. Outputs 'false'
+    var_export($time->isWithinNext('2 days'));
 
-    // Within 2 next weeks.
-    echo $time->isWithinNext('2 weeks');
+    // Within 2 next weeks. Outputs 'true'
+    var_export($time->isWithinNext('2 weeks'));
 
 .. php:method:: wasWithinLast($interval)
 
-You can also compare a ``Time`` instance within a range in the past::
+You can also compare a ``FrozenTime`` instance within a range in the past::
 
-    // Within past 2 days.
-    echo $time->wasWithinLast(2);
+    $time = new FrozenTime('-72 hours');
+    
+    // Within past 2 days. Outputs 'false'
+    var_export($time->wasWithinLast('2 days'));
 
-    // Within past 2 weeks.
-    echo $time->wasWithinLast('2 weeks');
+    // Within past 3 days. Outputs 'true'
+    var_export($time->wasWithinLast('3 days'));
+    
+    // Within past 2 weeks. Outputs 'true'
+    var_export($time->wasWithinLast('2 weeks'));
 
 .. end-time
 
-Dates
+FrozenDate
 =====
 
-.. php:class: Date
+.. php:class: FrozenDate
 
-The ``Date`` class in CakePHP implements the same API and methods as
-:php:class:`Cake\\I18n\\Time` does. The main difference between ``Time`` and
-``Date`` is that ``Date`` does not track time components, and is always in UTC.
+The immutable ``FrozenDate`` class in CakePHP implements the same API and methods as
+:php:class:`Cake\\I18n\\FrozenTime` does. The main difference between ``FrozenTime`` and
+``FrozenDate`` is that ``FrozenDate`` does not track time components, and is always in UTC.
```

Comment:
```suggestion
``FrozenDate`` is that ``FrozenDate`` does not track time components.
```
This is no longer true, but it's an implementation detail that shouldn't matter.

Reply:
ok, will remove it from next PR. Can't change it now cause already merged.